lib/qm-dsp/dsp/keydetection/GetKeyMode.cpp

```diff
@@ -182,7 +182,7 @@ int GetKeyMode::process(double *PCMData)
     // middle of its three bins :
     // Added 21.11.07 by Chris Sutton based on debugging with Katy
     // Noland + comparison with Matlab equivalent.
-    MathUtilities::circShift( m_ChrPointer, m_BPO, 1);
+    MathUtilities::circShift( m_ChrPointer, m_BPO, 2);

```

Comment:
The preceding comment now does no longer apply to the code.

Reply:
The magic shift by 2 is not obvious.

Reply:
Ok, I will update the comment. I guess the history is now irrelevant after 12 years.